locale/core.yml

```diff
@@ -410,19 +410,19 @@ core:
 
     # These translations are displayed as tooltips for discussion badges.
     badge:
-      hidden_tooltip: Hidden
+      hidden_tooltip: Schované
 
     # These translations are displayed as error messages.
     error:
       generic_message: "Jejda, něco se pokazilo. Obnov prosím stránku a zkus to znovu."
-      not_found_message: The requested resource was not found.
-      permission_denied_message: You do not have permission to do that.
-      rate_limit_exceeded_message: You're going a little too quickly. Please try again in a few seconds.
+      not_found_message: Požadovaný materiál nebyl nalezen
+      permission_denied_message: Na tuto operaci nemáte dostatečná práva.
+      rate_limit_exceeded_message: Jste moc rychlý, zpomalte
 
     # These translations are used as suffixes when abbreviating numbers.
     number_suffix:
-      kilo_text: K
-      mega_text: M
+      kilo_text: " tisíc"
```

Comment:
Hmm, nejsem si jistý kde se tohle používá, a jestli se to tam vejde. A taky může nastat problém se shodou, např. `3 tisíc`. Zatím to necháme a když na to někde narazím tak to upravím.